### ISSUE TITLE
Errors when running demo_toolbox

### ISSUE THREAD

**Issue by HeliumHead** [CorentinJ/Real-Time-Voice-Cloning#638]
Whenever I try to run "python .\demo_toolbox.py" I receive the following:

(voice-clone) C:\Users\\(anon)\Desktop\Real-Time-Voice-Cloning-master>python .\demo_toolbox.py
C:\Users\\(anon)\Desktop\Real-Time-Voice-Cloning-master\encoder\audio.py:13: UserWarning: Unable to import 'webrtcvad'. This package enables noise removal and is recommended.
  warn("Unable to import 'webrtcvad'. This package enables noise removal and is recommended.")
2021-01-23 18:00:41.115352: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'cudart64_100.dll'; dlerror: cudart64_100.dll not found
2021-01-23 18:00:41.118179: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
C:\Users\\(anon)\miniconda3\envs\voice-clone\lib\site-packages\umap\__init__.py:9: UserWarning: Tensorflow not installed; ParametricUMAP will be unavailable
  warn("Tensorflow not installed; ParametricUMAP will be unavailable")
Arguments:
    datasets_root:    None
    enc_models_dir:   encoder\saved_models
    syn_models_dir:   synthesizer\saved_models
    voc_models_dir:   vocoder\saved_models
    low_mem:          False
    seed:             None
    no_mp3_support:   False

Error: Model files not found. Follow these instructions to get and install the models:
https://github.com/CorentinJ/Real-Time-Voice-Cloning/wiki/Pretrained-models

The only thing I've changed is the directory to remove my name. Any questions or suggestions?

**Comment by blue-fish** [comment]
This is the key part of the error message:
```
Error: Model files not found. Follow these instructions to get and install the models:
https://github.com/CorentinJ/Real-Time-Voice-Cloning/wiki/Pretrained-models
```

**Comment by HeliumHead** [comment]
I've already installed the pretrained models, and double checked to make sure all the files were in the right directories, so I'm not sure what's up with that... And thanks for responding so quickly!

**Comment by blue-fish** [comment]
Could be a permissions issue with the files or the directories. Also make sure capitalization matches the wiki page.

Try launching python from the Real-Time-Voice-Cloning folder and see if the [model checks](https://github.com/CorentinJ/Real-Time-Voice-Cloning/blob/master/utils/modelutils.py) pass when manually run.

```
from pathlib import Path
Path("encoder\saved_models").is_dir()      # Should return True
Path("synthesizer\saved_models").is_dir()  # Should return True
Path("vocoder\saved_models").is_dir()      # Should return True
```

**Comment by HeliumHead** [comment]
Okay so I accidentally closed the environment and I made a new one pretty much the same way, but now I get these errors:

`(voice-clone) PS C:\Users\Wyatt\Desktop\Real-Time-Voice-Cloning-master> python .\demo_toolbox.py
C:\Users\Wyatt\Desktop\Real-Time-Voice-Cloning-master\encoder\audio.py:13: UserWarning: Unable to import 'webrtcvad'. This package enables noise removal and is recommended.
  warn("Unable to import 'webrtcvad'. This package enables noise removal and is recommended.")
2021-01-24 03:19:15.272827: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'cudart64_100.dll'; dlerror: cudart64_100.dll not found
2021-01-24 03:19:15.275594: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
C:\Users\Wyatt\miniconda3\envs\voice-clone\lib\site-packages\umap\__init__.py:9: UserWarning: Tensorflow not installed; ParametricUMAP will be unavailable
  warn("Tensorflow not installed; ParametricUMAP will be unavailable")
Arguments:
    datasets_root:    None
    enc_models_dir:   encoder\saved_models
    syn_models_dir:   synthesizer\saved_models
    voc_models_dir:   vocoder\saved_models
    low_mem:          False
    seed:             None
    no_mp3_support:   False

Librosa will be unable to open mp3 files if additional software is not installed.
Please install ffmpeg or add the '--no_mp3_support' option to proceed without support for mp3 files.
(voice-clone) PS C:\Users\Wyatt\Desktop\Real-Time-Voice-Cloning-master> print("hello world")
Unable to initialize device PRN
(voice-clone) PS C:\Users\Wyatt\Desktop\Real-Time-Voice-Cloning-master> python
Python 3.7.9 (default, Aug 31 2020, 17:10:11) [MSC v.1916 64 bit (AMD64)] :: Anaconda, Inc. on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from pathlib import Path
>>> Path("encoder\saved_models").is_dir()
True
>>> Path("synthesizer\saved_models").is_dir()
True
>>> Path("vocoder\saved_models").is_dir()
True
>>> exit()`

**Comment by HeliumHead** [comment]
Never mind, I just had to install a library
